### ISSUE TITLE
search doesn't work with context

### ISSUE THREAD

**Issue by yucer** [tinyerp/erppeek#81]
The context is necessary when searching objects by its name, because the ORM uses the value according to the language stated there.

But the execute function set the context to None for the search method: http://bit.ly/1VgmdVF

Is there a purpose for that ?

I thing using the search in another natural language is a basic function.

**Comment by yvaucher** [comment]
No idea this was done here when adding context key argument:

https://github.com/tinyerp/erppeek/commit/cc341868bb3abfed2a59dfe1ae4429d1d87c2999

**Comment by florentx** [comment]
Well, I've dropped `context` for `search` in a recent commit 0776800.

The new issue is that the order of arguments of search method has changed between Odoo 9 and Odoo 10.
```python
# Odoo 9 : openerp/models.py
    def search(self, cr, user, args, offset=0, limit=None, order=None, context=None, count=False):

# Odoo 10 : odoo/models.py
    def search(self, args, offset=0, limit=None, order=None, count=False):

```

To address your concern, I'll need to write more code.

**Comment by florentx** [comment]
Fixed in master branch with 5d0d591
